nominees/a11y.json

```diff
@@ -0,0 +1,35 @@
+{
+    "name": "a11yproject",
```

Comment:
According to the [website](https://www.a11yproject.com/), the name reads as `A11Y Project` instead of `a11yproject`. @SondosAhmed could you kindly update the name field. Thanks.